implementation/tests/test_dolfinx_cuas.py

```diff
@@ -0,0 +1,233 @@
+# Copyright (C) 2021 Sarah Roggendorf
+#
+# SPDX-License-Identifier:    MIT
+
+from dolfinx_contact.helpers import (epsilon, lame_parameters, sigma_func, R_minus)
+
+import dolfinx
+import dolfinx.io
+import dolfinx_cuas
+import dolfinx_cuas.cpp
+import numpy as np
+import ufl
+import pytest
+from mpi4py import MPI
+
+kt = dolfinx_cuas.cpp.contact.Kernel
+it = dolfinx.cpp.fem.IntegralType
+compare_matrices = dolfinx_cuas.utils.compare_matrices
+
+
+@pytest.mark.parametrize("theta", [1, -1, 0])
+@pytest.mark.parametrize("gamma", [10, 100, 1000])
+@pytest.mark.parametrize("dim", [2, 3])
+@ pytest.mark.parametrize("gap", [0.0, 0.02, -0.01])
+def test_contact_kernel(theta, gamma, dim, gap):
```

Comment:
It would be great if you could give a short description of what this test checks, as it is quite elaborate.